.github/workflows/cd.yml

```diff
@@ -38,19 +52,13 @@ jobs:
       - name: Test
         run: npm run test
 
-      - name: Set release
-        id: semrel
-        uses: go-semantic-release/action@v1
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          allow-initial-development-versions: true
-          force-bump-patch-version: true
-
-      - name: Publish npm package to Artifactory
+      - name: Publish package to npm
         run: |
-          export VERSION="${{ steps.semrel.outputs.version }}"
+          export VERSION="${{ needs.release.outputs.version }}"
```

Comment:
Is this right with needs. Not step.?

Reply:
Yup, following the same procedure as the client protos which successfully publish package to npm from the cd workflow.
https://github.com/momentohq/client_protos/blob/main/.github/workflows/push.yml#L61
I think I needed to add `needs: release` [here](https://github.com/momentohq/client-sdk-javascript/pull/50/files#diff-ea3ea8c9932adc7ba8161ceda844fedd43b006848ef1140c050cbd7ea0788a18R30)